games/i.yaml

```diff
@@ -63,10 +63,10 @@
   content: open
   originals:
   - Ace of Spades
-  repo: https://github.com/iamgreaser/iceball
+  repo: https://github.com/arsdragonfly/iceball
```

Comment:
I'm not familiar with this project so what's the story here? Is this fork hostile or officially sanctioned?